.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: node_js
+node_js:
+  - "8"
```

Comment:
Node 8 support was recently dropped elsewhere, is it needed here?

Reply:
Not needed.  I don't think we need to actively remove that testing where it still works, but I'm ok not adding it to new packages.  Removing.